csi.proto

```diff
@@ -78,17 +118,29 @@ service Node {
   rpc NodeGetVolumeStats (NodeGetVolumeStatsRequest)
     returns (NodeGetVolumeStatsResponse) {}
 
-
   rpc NodeExpandVolume(NodeExpandVolumeRequest)
     returns (NodeExpandVolumeResponse) {}
 
-
   rpc NodeGetCapabilities (NodeGetCapabilitiesRequest)
     returns (NodeGetCapabilitiesResponse) {}
 
   rpc NodeGetInfo (NodeGetInfoRequest)
     returns (NodeGetInfoResponse) {}
 }
+
+service ObjectStore {
+  option (alpha_service) = true;
+
+  rpc ObjectStoreRead(ObjectStoreReadRequest)
+    returns (ObjectStoreReadResponse) {
+      option (alpha_method) = true;
```

Comment:
it's arguable that marking the method as alpha is redundant because the entire service is already alpha.